modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ServerConfigurator.java

```diff
@@ -321,7 +365,10 @@ public void checkPaths() throws ConfigurationException {
             log.warn("NXP-8014 Packages cache location changed. You can safely delete {} or move its content to {}",
                     () -> oldPackagesPath, this::getPackagesDir);
         }
-
+        File oldPath = new File(getRuntimeHome(), "data" + File.separator + "vcsh2repo");
+        String message = String.format("NXP-5370, NXP-5460. " + "Please rename 'vcsh2repo' directory from %s to %s",
+                oldPath, new File(generator.getDataDir(), "h2" + File.separator + "nuxeo"));
```

Comment:
This is really obsolete (I can't even remember when it was used). I don't think it's worth keeping.

Reply:
I removed it.